sdk/@launchdarkly/observability-react-native/package.json

```diff
@@ -8,6 +8,7 @@
 	},
 	"scripts": {
 		"build": "vite build",
+		"docs": "typedoc",
```

Comment:
Does this automatically use the config, or does it need `typedoc --options ./typedoc.json"`?